IPv6 support is broken and is forced to be disabled.
This place is sufficient - IPv6 initialisation will now cause an error.

--- a/src/libircclient.c	2011-05-26 21:23:06.000000000 +0200
+++ b/src/libircclient.c	2011-05-26 21:23:33.000000000 +0200
@@ -194,7 +194,7 @@
 			const char * username,
 			const char * realname)
 {
-#if defined (ENABLE_IPV6)
+#if 0
 	struct sockaddr_in6 saddr;
 	struct addrinfo ainfo, *res = NULL;
 	char portStr[32];
